### PR TITLE
Platform 9328 publish

### PR DESCRIPTION
**Pull request by abador**

### CODE REVIEW

File: JenkinsFile-release.groovy [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        if (params.dry_run) {
+          print("yarn pub")
+        } else {
+          sh("yarn pub")
+        }
+        FAILED_STAGE = 'publish'
+            }
+          }
+        }
+
```
Comment (adpaste) [inline JenkinsFile-release.groovy:118]: Indentation looks bad in the code below.

**Review by adpaste — APPROVED** [review]

File: JenkinsFile-release.groovy [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        stage('publish') {
+          steps {
+            script {
+        if (params.dry_run) {
+          print("yarn pub")
+        } else {
+          sh("yarn pub")
+        }
```
Comment (abador) [inline JenkinsFile-release.groovy:113]: remove dry run

Comment (abador) [inline JenkinsFile-release.groovy:113]:
```suggestion
              sh("yarn pub")
```

File: JenkinsFile-release.groovy [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            echo "Jira issue key: ${issueKey}"
+            }
+          }
+        }
+
+        stage('publish') {
+          steps {
+            script {
+              sh("yarn pub")
+        FAILED_STAGE = 'publish'
```
Comment (abador) [inline JenkinsFile-release.groovy:110]:
```suggestion
              FAILED_STAGE = 'publish'
```

**Pull request merged**